kubemarine/templates/plugins/nginx-ingress-controller-v1.4.yaml.j2

```diff
@@ -0,0 +1,697 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    app.kubernetes.io/instance: ingress-nginx
+    app.kubernetes.io/name: ingress-nginx
+    {% if rbac['admission'] == "pss" and rbac["pss"]["pod-security"] == "enabled" and rbac["pss"]["defaults"]["enforce"] != "privileged" %}
+    pod-security.kubernetes.io/enforce: privileged
+    pod-security.kubernetes.io/enforce-version: latest
+    pod-security.kubernetes.io/audit: privileged
+    pod-security.kubernetes.io/audit-version: latest
+    pod-security.kubernetes.io/warn: privileged
+    pod-security.kubernetes.io/warn-version: latest
+    {% endif %}
+  name: ingress-nginx
+---
+apiVersion: v1
+automountServiceAccountToken: true
+kind: ServiceAccount
+metadata:
+  labels:
+    app.kubernetes.io/component: controller
+    app.kubernetes.io/instance: ingress-nginx
+    app.kubernetes.io/name: ingress-nginx
+    app.kubernetes.io/part-of: ingress-nginx
+    app.kubernetes.io/version: 1.4.0
+  name: ingress-nginx
+  namespace: ingress-nginx
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app.kubernetes.io/component: admission-webhook
+    app.kubernetes.io/instance: ingress-nginx
+    app.kubernetes.io/name: ingress-nginx
+    app.kubernetes.io/part-of: ingress-nginx
+    app.kubernetes.io/version: 1.4.0
+  name: ingress-nginx-admission
+  namespace: ingress-nginx
```

Comment:
Looks like previously we intentionally ignored everything related to webhook.
Compare [original](https://github.com/kubernetes/ingress-nginx/blob/controller-v1.2.0/deploy/static/provider/do/deploy.yaml) and [our template](https://github.com/Netcracker/KubeMarine/blob/0.8.0/kubemarine/templates/plugins/nginx-ingress-controller-v1.2.yaml.j2#L29). I would suggest to keep this approach and if we need to include new feature we could do this intentionally within separate PR

Reply:
Could you remind me what the point was? I didn't find any information.

Reply:
Do you mean to remind why we cut off webhook previously? Personally, no idea.

Reply:
As we decided at the meeting, webhook should be included into new version of `nginx-ingress-controller`